config/config.html

```diff
@@ -5,6 +5,10 @@
 <input ng-model="config.docker_host" placeholder="Docker remote host (defaults to env var DOCKER_HOST)" class="input-block-level input-xxlarge" type="text">
 <input ng-model="config.image" placeholder="Docker image to use. Default: strider/strider-docker-slave" class="input-block-level input-xxlarge" type="text">
 <input ng-model="config.dns" placeholder="comma separated list of dns servers, default: 8.8.8.8" class="input-block-level input-xxlarge" type="text">
+<p>
+  Use the following to bind host directories to volumes in a docker container:
+</p>
+<input ng-model="config.docker_volumeBinds" placeholder="comma seperated list of bindings e.g.: /some/host/dir:/in_container:rw,/other:/foo:rw, Default: ''" class="input-block-level input-xxlarge" type="text">
 <label class="checkbox">
```

Comment:
@knownasilya addressed in new commit.

BTW: what is the common practice around here regarding commits which are fixes for
issues discovered during review? 
I normally add an extra commit which addresses the discovered issues. I usually mark the extra commit as something that should be squashed into a previous commit before merging.